eslintrc

```diff
@@ -1,155 +0,0 @@
-# ESLint documentation can be found at http://eslint.org/docs/
```

Comment:
Oh, I haven't been using that because it runs on all files and spews too much. But yeah, we should either fix it or remove it. How about we remove the "-c" option to make it use the ~/.eslintrc file?